packages/wonder-blocks-data/util/request-handler.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import type {IRequestHandler} from "./types.js";
+
+/**
+ * Base implementation for creating a request handler.
+ *
+ * Provides a base implementation of the `IRequestHandler` base class for
+ * use with the Wonder Blocks Data framework.
+ */
+export default class RequestHandler<TOptions, TData>
+    implements IRequestHandler<TOptions, TData> {
+    _type: string;
+
+    constructor(type: string) {
+        this._type = type;
+    }
+
+    get type() {
+        return this._type;
+    }
+
+    invalidateCache(options: TOptions): boolean {
+        /**
+         * By default, the cache is always valid.
+         */
+        return false;
+    }
```

Comment:
Is there anytime a sub-claass of RequestHandler would also want to return `false`?

Reply:
Not sure I understand the question. Can you rephrase it?

Reply:
I've updated the `request-handler.md` to cover this method and its uses.